pkg/machine/ignition.go

```diff
@@ -161,6 +195,22 @@ func getFiles(usrName string) []File {
 	var (
 		files []File
 	)
+
+	lingerExample := `[Unit]
+Description=A systemd user unit demo
+After=network-online.target
+Wants=network-online.target podman.socket
```

Comment:
Not necessarily for this PR, but note that `network-online.target` doesn't work for user units.. See 

- https://github.com/containers/podman/issues/10655#issuecomment-900604163
- https://github.com/systemd/systemd/issues/3312#issuecomment-893009485